### PR TITLE
Document additional relation types for process metadata #531

### LINKED ISSUE #531: License for UDP?
A license field for user-defined processes might be a good idea.

### PR DESCRIPTION
**Pull request by m-mohr**
Closes #531

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -5262,11 +5262,19 @@ components:
             (link to older version) and `successor-version` (link to newer version) can also be used
             to show the relation between versions.
 
+            2. `version-history`: A link to a changelog and/or a list of versions of the process
```
Comment (soxofaan) [inline openapi.yaml:5265]: will this duplicate  `2.` in the markdown give duplicated rendering of the "2" bullet?

**Review by soxofaan — APPROVED** [review]

**Comment by soxofaan** [comment]
I think this PR is fine to merge as-is already

however by looking at it I'm a bit confused by this existing bit:

> latest-version: If a process has been marked as deprecated, a link SHOULD point to the preferred version of the process. The relation types predecessor-version (link to older version) and successor-version (link to newer version) can also be used to show the relation between versions.

How is this supposed to work? There is no version concept (yet) for UDPs, nor do they have a public URLs to link to .  

UDPs are only by defined by process id, so does this  documentation suggest to put the version in the the process id, e.g. `my_evi_v2`, `my_evi_v3`? (Note that dots are not allowed in process id, so this might get ugly)

File: openapi.yaml (hunk 1 above)
Comment (m-mohr) [reply openapi.yaml:5265]: Good point, fixed.

**Comment by m-mohr** [comment]
The versioning was meant more for informative purposes, e.g. for tracability, and not necessarily lives within the API. The ID should usually be the same and old versions should not be usable anymore. At least right now. We could also integrate something on the API level, but that is future work, I think.

**Pull request merged**